cps/web.py

```diff
@@ -3883,3 +3944,214 @@ def convert_bookformat(book_id):
     else:
         flash(_(u"There was an error converting this book: %(res)s", res=rtn), category="error")
     return redirect(request.environ["HTTP_REFERER"])
+
+
+def register_oauth_blueprint(blueprint):
```

Comment:
Now that we are using blueprints - could we try to move the new features into their own modules?  Just trying to minimize the amount of stuff we keep putting into web.py.

Reply:
This function is just record which oauth is used for calibre web.
We will check for oauth settings in sessions, we need know how many oauth is enabled in calibre web.